src/examples/genesis/api_claim_creator_rewards/cli.sh

```
@@ -0,0 +1,9 @@
+# Claim creator rewards across all your bonding-curve and Raydium buckets.
+# Defaults the wallet to the configured signer; network is auto-detected from RPC.
+mplx genesis claim-creator-rewards
+
+# Claim on behalf of a different creator fee wallet (you still pay fees).
+mplx genesis claim-creator-rewards --wallet <CREATOR_FEE_WALLET>
```

Shellcheck (0.11.0)

[info] The mentioned syntax error was in this simple command.

(SC1009)